site/package.json

```diff
@@ -4,17 +4,16 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "yarn build-blocks && yarn generate-blocks-data && yarn generate-sitemap && next build",
+    "build": "yarn build-blocks && yarn exe scripts/generate-blocks-data && yarn exe scripts/generate-sitemap && yarn exe scripts/create-db-indexes && next build",
     "build-block": "./scripts/build-blocks.sh",
     "build-blocks": "find ../hub -type f | xargs ./scripts/build-blocks.sh",
     "clean": "rimraf ./.next/",
-    "dev": "yarn generate-sitemap && yarn generate-blocks-data && next dev",
+    "dev": "yarn exe scripts/generate-sitemap && yarn exe scripts/generate-blocks-data && yarn exe scripts/create-db-indexes && next dev",
```

Comment:
We can try composing scripts outside `package.json` as this is more flexible. I can try one small idea in a follow-up once you’ve merged.